<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="MAINWINDOW">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>220</y>
          <width>441</width>
          <height>251</height>
         </rect>
        </property>
        <property name="title">
         <string>Login</string>
        </property>
        <widget class="QLineEdit" name="username">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>80</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="password">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>130</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>190</y>
           <width>112</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>190</y>
           <width>112</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="adminLogin"/>
      <widget class="QWidget" name="managerLogin">
       <widget class="QTableView" name="managerTable">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>12</y>
          <width>751</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="viewItems">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>500</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>View Items</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
